proposals/relaxed-simd/Overview.md

```diff
@@ -182,17 +182,17 @@ where:
 - the intermediate `b * c` is be rounded first, and the final result rounded again (for a total of 2 roundings), or
 - the the entire expression evaluated with higher precision and then only rounded once (if supported by hardware).
 
-### Relaxed bitselect (blend/laneselect)
+### Relaxed laneselect
```

Comment:
I suggest just "laneselect", without "relaxed"

Reply:
The relaxed isn't really part of the name, plus the names aren't confirmed. I can file an issue for this. I think the instruction names should reflect this non-determinism, it doesn't have to be "relaxed".